ui/src/App.vue

```diff
@@ -51,11 +51,36 @@ export default {
   data: () => ({
     individuals: {
       entities: []
-    }
+    },
+    page: 1,
+    numPages: 0
   }),
   async created() {
-    let response = await getIndividuals(this.$apollo);
+    let response = await getIndividuals(this.$apollo, 1, 50);
     this.individuals = response.data.individuals;
+    this.numPages = this.individuals.pageInfo.numPages;
+  },
+  mounted() {
+    this.scroll();
+  },
+  methods: {
+    scroll() {
+      window.onscroll = async () => {
+        let bottomOfWindow =
+          document.documentElement.scrollTop + window.innerHeight ===
+          document.documentElement.offsetHeight;
+
+        if (bottomOfWindow && this.page !== this.numPages) {
+          this.page++;
+          let response = await getIndividuals(this.$apollo, this.page, 50);
+          this.individuals.entities = [
+            ...this.individuals.entities,
+            ...response.data.individuals.entities
+          ];
+          this.numPages = this.individuals.pageInfo.numPages;
+        }
```

Comment:
Shouldn't it be better to have this in the component instead of on the app?

I think the code that expands the page should be part of the component. The component will receive a method to get the identities (or an iterator) and each time it needs to expand, it calls that function.

What do you think?

Reply:
It makes sense, I'll update the PR!

Reply:
I commented on the issue, please check it when you can @sduenas